packages/tldraw/src/lib/shapes/geo/cloudOutline.ts

```diff
@@ -319,7 +326,7 @@ export function pointsOnArc(
 	const startAngle = Vec2d.Angle(center, startPoint)
 	const endAngle = Vec2d.Angle(center, endPoint)
 
-	const l = shortAngleDist(startAngle, endAngle)
+	const l = clockwiseAngleDist(startAngle, endAngle)
```

Comment:
sometimes, on the very corners of the cloud shapes, I was seeing the outline being calculated using the anticlockwise distance for some arcs, since the outer arc got too long, so I've added a helper for calculating the clockwise angle dist explicitly.